.github/workflows/regression_ci.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+name: Performance Regression Test
+
+on:
+  pull_request:
```

Comment:
Looking good. Just one thought, should we disable the job when the regression tests themselves are changed? That way we wouldn't have the meaningless failure

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-excluding-paths

Reply:
That's neat, I think we should disable when changes are made to the tests or else it's likely they won't behave as expected. I'll add that in the revision

Reply:
The check still runs on this PR because all file paths changed in the PR have to match those in `paths-ignore` in order for it to be ignored. This PR changes a file outside of the `tests/regression` crate so the check remains enabled. I don't think we should include the `.github/workflows.regression_ci.yml` path in paths-ignore because the only way to test if changes to the runner work as intended are for it to run in the pull request.